### DISCUSSION TITLE
Threat Blocked: Trojan:Script/Wacatac.B!ml

### DISCUSSION THREAD

**Discussion by trevaccinate** [smasherprog/EqTool#123] [Q&A]
I tried to download any version of EqTool for p99 and it gets blocked for virus.
Tried Chrome, Edge, and Firefox. All blocked.

Checked my virus logs and it comes up with Trojan:Script/Wacatac.B!ml
What's up with that?

**Comment by smasherprog** [comment]
Its because its not signed with a code signing certificate most likely..

**Reply by trevaccinate** [reply]
So what do I have to do to be able to download?

**Reply by princealexi25** [reply]
I am able to download, but the program is eventually blocked and removed.  I had to set an exemption of the file folder, which I'm not happy about.  I may switch to nParse/GINA.  

So they tell you to lower your security, then a malicious coder puts something into a push?  Not smart.

Although you might have a false positive there.. I get unsigned errors.

**Comment by smasherprog** [comment]
Going to close this as it should be fixed now. The exe is signed, which helps with the anti virus scanners freaking out .. I think :)
Please create a new issue if the problem persists
